docs/robot_playwright.rst

```diff
@@ -0,0 +1,164 @@
+
+* it verifies that Node.js has been installed
+* it downloads and installs the Browser keyword library
+* it downloads and installs the Node.js modules and drivers for
+  playwright.
+
+.. note::
+
+   The installation of playwright contains drivers for all supported
+   browsers, so there's no need to manually install drivers such as
+   chromedriver. It works right out of the box!
+
+To run the script you need to make sure your working directory is at
+the root of your repository. Then, run the following command:
+
+.. code-block:: console
+
+    $ cci robot install_playwright
+
+.. tip::
+
+   You can use the ``--dry_run`` (or ``-n``) option to see what the
+   command will do without actually installing anything.
+
+
+Running an example test
+-----------------------
+
+As mentioned earlier, this is an experimental release of playwright
+integration, so any CumulusCI keywords that rely on selenium won't
+work. However, the following example shows off how easy it can be to
+write playwright-based test with off-the-shelf `keywords provided by
+the Browser library
+<https://marketsquare.github.io/robotframework-browser/Browser.html>`_
+
+.. code-block:: robotframework
+
+    *** Settings ***
+    Resource     cumulusci/robotframework/Salesforce.robot
+    Library      Browser
+
+    Suite Setup  Open test browser
+
+    *** Keywords ***
+    Open test browser
+        New Browser  chromium  headless=false
+        ${url}=      Login URL
+        New Page     ${url}
+        Wait until network is idle
+
+    *** Test Cases ***
+
+    Go to user profile
+        Click    button:has-text("View profile")
+        Click    .profile-card-name .profile-link-label
+
+        Wait until network is idle
+        Take screenshot
+
+    Go to contacts home
+        Click            button:has-text("App Launcher")
+        Fill text        input[placeholder='Search apps and items...']  Contacts
+        Click            one-app-launcher-menu-item:has-text("Contacts")
+
+        Wait until network is idle
+        Take screenshot
+
+To run the test, save the above code in a .robot file (eg:
+example.robot) and then run it with the standard robot task:
+
+.. code-block:: console
+
+    $ cci task run robot --suites example.robot
+
+
+Things to notice
+^^^^^^^^^^^^^^^^
+
+This example test is not able to use any of the existing
+selenium-based keywords. For that reason, this test creates
+a new ``Open Test Browser`` that uses the Browser keywords
+`New Browser
+<https://marketsquare.github.io/robotframework-browser/Browser.html#New%20Browser>`_
+and `New Page
+<https://marketsquare.github.io/robotframework-browser/Browser.html#New%20Page>`_
+to open the browser.
+
+This test also uses the Browser keyword
+`Wait until network is idle
+<https://marketsquare.github.io/robotframework-browser/Browser.html#Wait%20Until%20Network%20Is%20Idle>`_
+before taking a screenshot. This is a convenient keyword that usually
```

Comment:
What determines whether it really waits? What does "usually" mean?

Reply:
Weird. I don't know why I wrote "usually" there. It will always wait. <scratches head>. 

How about this?

> This is a keyword that
waits for there to be at least one instance of 500ms of no network
traffic on the page after it starts to load. This seems to be more reliable
and easier to use method than waiting for a page-specific element to
appear.